src/components/withToggleVisibilityView.tsx

```diff
@@ -1,19 +1,18 @@
 import type {ComponentType, ForwardedRef, ReactElement, RefAttributes} from 'react';
 import React from 'react';
 import {View} from 'react-native';
-import type {SetOptional} from 'type-fest';
```

Comment:
These changes were only necessary because I upgraded `type-fest` and a bug fix in that repo exposed existing issues with the types in this code.